tests/conformance/scripts/run-tests/runners/RustWriterTestRunner.ts

```diff
@@ -0,0 +1,35 @@
+import { exec } from "child_process";
+import { intersection } from "lodash";
+import { join } from "path";
+import { promisify } from "util";
+import { TestVariant } from "variants/types";
+
+import { WriteTestRunner } from "./TestRunner";
+
+export default class RustWriterTestRunner extends WriteTestRunner {
+  readonly name = "rust-writer";
+
+  async runWriteTest(filePath: string): Promise<Uint8Array> {
+    const { stdout, stderr } = await promisify(exec)(`./conformance_writer ${filePath}`, {
+      cwd: join(__dirname, "../../../../../rust/target/debug/examples"),
+      encoding: undefined,
+    });
+
+    if (stderr instanceof Buffer) {
+      const errText = new TextDecoder().decode(stderr);
+      if (errText.length > 0) {
+        console.error(errText);
+      }
+    }
+    return stdout as unknown as Uint8Array;
+  }
+
+  supportsVariant(variant: TestVariant): boolean {
+    return (
+      intersection(
+        [...variant.features],
+        ["ch", "mx", "st", "rsh", "rch", "ax", "mdx", "chx", "sum", "pad"],
+      ).length === 0
+    );
+  }
```

Comment:
I don't see the point of doing this now, because you aren't running this test in CI anyway. Maybe just return true for now?

In general we should use the actual enum to make this more readable, e.g. some of the other runners use things like `variant.features.has(TestFeatures.AddExtraDataToRecords)`.

Also, if you do want to keep this behavior, it could've been written as `return variant.features.size === 0`.